templates/antora/namespace.handlebars

```diff
@@ -0,0 +1,19 @@
+= {{title}}
```

Comment:
TODO: we can clean up the handlebars in this branch if required, it requires more cleanup.  Leaving here for now incase we need to do somthing for namespace index pages, right now clicking on the navigation just drops down the menu without a page change.